### PR TITLE
misc updates, mostly related to user-facing strings

### LINKED ISSUE #1890: wording on facility settings is incorrect
phrasing "Allow users to edit ...":

image

Incorrect for admins, who can always edit this info (including their own):

image

We should clarify the facility settings, as e.g. "Allow learners and coaches to edit ..."

### LINKED ISSUE #2100: update auth message translation strings
Translation strings should not be composed programmatically. Instead, they should either be self-contained or constructed programmatically using ICU syntax.

see: https://github.com/learningequality/kolibri/pull/1473#discussion_r116873811

While we're working on this, we should audit the messages themselves: https://github.com/learningequality/kolibri/pull/1473#discussion_r116874856

we need to check in on what the right way to refer to user types is (e.g. using capitalization, single quotes, or just plain words)

### LINKED ISSUE #2269: 'welcome' modal is not fully keyboard-accessible
After going through the onboarding process, 'enter' and 'escape' should both close the modal without needing to tab to the 'ok' button.

### LINKED ISSUE #2277: rename facility configuration -> settings
because it's a shorter word

### LINKED ISSUE #2380: Close/Back modal buttons do not have visual focus outline
## Summary

Buttons are focusable, and modals can be closed with the `Enter` key once the `X` button is in focus, it's is just that there is no visual indication that focus actually grabbed it.

Modals checked:
- Welcome to Kolibri!
- Add new account
- Add new class
- Import - choose the source

## System information

 - Version: `kolibri-0.6.dev020171007224512-git.pex`
 - Operating system: Ubuntu 17.04
 - Browser: Chrome

### LINKED ISSUE #2635: button is centered profile page
### Observed behavior

In the 'profile' page, the 'save' button is centered, but our style guide specifies left-aligned. 

image

### Expected behavior

standardized

### PR DESCRIPTION
**Pull request by indirectlylit**
# Details

### Summary

* fixes #2100
* fixes #2277
* fixes #1890
* fixes #2635
* fixes #2380
* fixes #2269

# Contributor Checklist

- [x] PR has the correct target milestone
- [x] PR has the appropriate labels
- [x] Changes in the PR do not introduce accessibility regressions ([confimed by testing with one of the recommended tools](http://kolibri.readthedocs.io/en/develop/dev/manual_testing.html#accessibility-a11y-testing)) 
- [x] If PR is ready for review, it has been assigned or requests review from someone
- [ ] Documentation is updated as necessary
- [ ] External dependency files are updated (`yarn` and `pip`)
- [ ] If internal dependency is updated, link to diff is included
- [ ] Screenshots of any front-end changes are in the PR description
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] You've added yourself to AUTHORS.rst if you're not there

# Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] PR has been fully tested manually

### CODE REVIEW

**Comment by indirectlylit** [comment]
(Do not merge yet - I'll incorporate #2638 first)

File: kolibri/core/assets/src/views/auth-message.vue [hunk 1]
```diff
@@ -36,19 +36,16 @@
...
       },
     },
     $trs: {
-      admin: 'an Admin',
-      adminOrCoach: 'an Admin or Coach',
+      admin: 'You must be signed in as an Admin to view this page',
+      adminOrCoach: 'You must be signed in as an Admin or Coach to view this page',
+      learner: 'You must be signed in as a Learner to view this page',
+      registeredUser: 'You must be signed in to view this page',
+      superuser: 'You must be a Superuser to view this page',
```
Comment (christianmemije) [inline kolibri/core/assets/src/views/auth-message.vue]: What about 'You must have Superuser permissions to view this page'?

File: kolibri/plugins/management/assets/src/views/top-nav/index.vue [hunk 2]
```diff
@@ -40,7 +40,7 @@
     $trs: {
       classes: 'Classes',
       data: 'Data',
-      configuration: 'Configuration',
+      settings: 'Settings',
```
Comment (christianmemije) [inline kolibri/plugins/management/assets/src/views/top-nav/index.vue:43]: We should also update the page title, this page's h1, and the route.

localhost_8000_management_facility_

File: kolibri/core/assets/src/views/auth-message.vue (hunk 1 above)
Comment (radinamatic) [reply kolibri/core/assets/src/views/auth-message.vue]: I second the motion!

**Comment by indirectlylit** [comment]
@christianmemije feedback addressed

**Pull request merged**